modules/account-iam-resources/versions.tf

```diff
@@ -7,7 +7,7 @@ terraform {
       version = ">= 4.0"
     }
     rhcs = {
-      version = ">= 1.1.0"
+      version = "1.4.0-prerelease.3"
```

Comment:
I guess it should be 1.4.0

Reply:
I will change it to 1.4.0 in different commit after release